db/schema.rb

```diff
@@ -24,6 +24,15 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "actor_types_taxonomies", force: :cascade do |t|
+    t.bigint "actor_type_id", null: false
+    t.bigint "taxonomy_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```

Comment:
updated_at not required (as per https://github.com/dumparkltd/marine-defrag-server/issues/23#issuecomment-946675036) but I also ok to keep it if it doesn't hurt